extensions-core/s3-extensions/pom.xml

```diff
@@ -116,6 +116,11 @@
       <artifactId>aws-java-sdk-sts</artifactId>
       <version>${aws.sdk.version}</version>
     </dependency>
+    <dependency>
+      <groupId>it.unimi.dsi</groupId>
+      <artifactId>fastutil-core</artifactId>
+      <scope>provided</scope>
```

Comment:
Provided means that, at run time, in production, this jar will already be on the class path. Is that true in this case? Does core Druid or S3 already include this jar? If not, the scope has to be the default so that the jar is downloaded and put on the class path.

Reply:
Yeah Druid processing already has this jar.